src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -916,6 +916,10 @@ protected override void ProcessRecord()
                         }
 
                         result = SessionState.Path.SetLocation(Path, CmdletProviderContext);
+#if UNIX
+                        // Be consistent with most Unix shells that set the environment variable 'OLDPWD' when changing location
+                        Environment.SetEnvironmentVariable("OLDPWD", Path);
```

Comment:
I don't see the point - we don't use it anywhere.

Reply:
Agreed - I also don't see the point. Maybe if `ps1` files ran in a new process, then it might make sense, but they don't.

And if setting the environment variable was useful, we should explain why it is UNIX only.

Reply:
Ok, I've removed it. I found out later that on Linux `cd` is also an alias for `Set-Location` (my assumption was that native Linux commands are not overridden by pwsh but this seems to be one of the exceptions)